crates/sui-graphql-e2e-tests/tests/stable/transactions/filters/transaction_ids.move

```diff
@@ -66,7 +66,7 @@ module Test::M1 {
 
 //# run-graphql
 {
-  transactionBlocks(filter: {recvAddress: "@{A}" transactionIds: []}) {
+  transactionBlocks(filter: {affectedAddress: "@{A}" transactionIds: []}) {
```

Comment:
shouldnt these cause changes in the `.exp`?

Reply:
i suppose not because senders will always show up in recipients

Reply:
(Not always, but most of the time, yes).